modules/HomePage/components/TaskList/TaskList.tsx

```diff
@@ -104,7 +109,7 @@ const TaskList: React.FC<TaskListProps> = ({
         </Group>
       </Modal>
 
-      {sortedTasks.map((task) => (
+      {todos?.map((task) => (
```

Comment:
What if `todos.length === 0`? Wouldn't an error be thrown inside the map loop?

Reply:
Are you saying, if there are no todo item in the db, I might get an error here with this implementation? How I am understanding is, if todos.length ===0, that means there are no items in that array, ie no todos in the db. So no todos would be rendered.